docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Use ML.NET in a TensorFlow transfer learning scenario for image classification
+description: Discover how to use ML.NET in a TensorFlow transfer learning scenario to classify images by reusing a pre-trained TensorFlow model.
+ms.date: 03/20/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to reuse a pre-trained TensorFlow model with ML.NET so that I can classify images with a small amount of training data.
+---
+# Tutorial: Use ML.NET in a TensorFlow transfer learning scenario for image classification
+
+This sample tutorial illustrates using ML.NET to reuse a pre-trained TensorFlow model to classify images with a small amount of training data via a .NET Core console application using C# in Visual Studio 2017. In the world of machine learning, this is called transfer learning.
+
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+
+This tutorial and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning GitHub repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Understand the problem
+> * Build and Train the model
+> * Predict with the trained model
+> * Deploy and Predict with a loaded model
+
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/TransferLearningTF) repository.
+
+## Understand the problem
+
+Image classification is a common Machine Learning problem which has well-known solutions. This tutorial uses an approach that mixes new techniques (deep learning) and traditional (Logistic Regression) techniques.
+
+This model uses the [Inception model](https://storage.googleapis.com/download.tensorflow.org/models/inception5h.zip) as a *featurizer* (the model is already stored in the [assets folder](./ImageClassification.Train/assets/inputs/inception/)). This means that the model processes input images through the neural network, using the tensor output before classification. This tensor contains the *image features*, used for image identification.
```

Comment:
Provide some detail on what the pre-trained model InceptionV3 is. What is the default setup of InceptionV3, different categories it supports and how and why are we trying to customize it.